hasjob/templates/detail.html

```diff
@@ -420,9 +420,10 @@
   });
   $(function() {
     $('.sidebar-pstar').click(function(e) {
-      $('.sidebar-pstar .pstar').click();
+      if(e.target == this || $(e.target).is('.pstar-caption')) {
+        $(this).find('.pstar').trigger('click');
```

Comment:
I suggest we avoid triggering explicit clicks (the dropdown bug was due to this).